### ISSUE TITLE
Master: error clicking on project when not signed in

### ISSUE THREAD

**Issue by brec9824** [uidaho/squireproject#85]
If you click on a project when not signed in you get redirected to an error page. Here is what it looks like: 
![error1](https://cloud.githubusercontent.com/assets/16866862/14589792/9ab3290c-049f-11e6-902f-36b73b348d61.PNG)

**Comment by brandonjank** [comment]
@uidaho/lambda 

It needs to handle authentication, it should be done at a controller level if possible.

If you want to require login for every page in the controller, insert this method:

```
    /**
     * Runs before every method in the class, useful for what's below.
     */
    public function __construct()
    {
        /**
         * This will require login for the entire controller and then return
         * the user back where they were going.
         */
        $this->middleware('auth');
    }
```

If you want a method to require authentication just for itself, use this at the top:

```
if (!Auth::check()) {
    return redirect('/login');
}
```

If you use the auth middleware, the user will automatically be returned to the original page after authenticating.

**Comment by brec9824** [comment]
Fixed in pull #91
